artist-block-backend-netcore-services/account-service-tests/Controller/UserRegistrationControllerTests/UserRegistrationControllerTests.cs

```diff
@@ -43,6 +44,7 @@ public class UserRegistrationControllerTests
         Bio = "Very experienced at painting stuff",
         Location = "Congo",
         YearsOfExperience = "23+",
+        AddPainterSpecialityDtos = new List<AddPainterSpecialityDto>(),
```

Comment:
Should we make as an empty List? Or create an `expectedSpeciality` to compare with?